templates/apps/wry/src/lib.rs.hbs

```diff
@@ -109,11 +122,6 @@ fn main() -> Result<()> {
 
                 Ok(Response::builder().header(CONTENT_TYPE, meta).body(data.into())?)
             }
-
-            #[cfg(target_os = "android")]
```

Comment:
seems this piece is still required for android, I can't run android without that

Reply:
yeah right, thanks for letting us know, I will update it shortly

Reply:
should be good now